privacy-dashboard/privacy-dashboard-impl/src/main/java/com/duckduckgo/privacy/dashboard/impl/ui/PrivacyDashboardRenderer.kt

```diff
@@ -37,11 +37,16 @@ class PrivacyDashboardRenderer(
     private val onOpenSettings: (String) -> Unit,
     private val onClose: () -> Unit,
     private val onSubmitBrokenSiteReport: (String) -> Unit,
+    private val onGetToggleReportOptions: () -> Unit,
+    private val onSendToggleReport: () -> Unit,
+    private val onRejectToggleReport: () -> Unit,
+    private val onSeeWhatIsSent: () -> Unit,
+    private val onShowNativeFeedback: () -> Unit,
 ) {
 
     private var lastSeenPrivacyDashboardViewState: ViewState? = null
 
-    fun loadDashboard(webView: WebView, initialScreen: InitialScreen) {
+    fun loadDashboard(webView: WebView, initialScreen: InitialScreen, toggleOpener: String) {
```

Comment:
nit: looks like there is a set of predefined values for `toggleOpener` - it could be an enum instead of a string.